### DISCUSSION TITLE
[question] add `lint` script

### DISCUSSION THREAD

**Discussion by cloud-walker** [mswjs/data#100] [Ideas]
@marcosvega91 @kettanaito 

I was wondering if adding a base eslint configuration with a `lint` script would be appreciated.

If so, I would be happy to start a PR about it!

**Comment by kettanaito** [comment]
Hey, @cloud-walker. That'd be a great addition to the project! Would be thankful for a pull request.

**Comment by kettanaito** [comment]
## Update

I've added the `lint` script that focuses on linting the _packaged package_ instead of linting the source code:

https://github.com/mswjs/data/blob/471a1f6154103a50eedd61f938fb2c22cd9a3af7/package.json#L10
